tardis/tests/tests_slow/conftest.py

```diff
@@ -1,9 +1,85 @@
 import os
+import shutil
+import tempfile
+import yaml
 import numpy as np
 import pytest
 from astropy import units as u
+from astropy.tests.helper import remote_data
 import tardis
 
+# For specifying error while exception handling
+from socket import gaierror
+
+try:
+    import dokuwiki
+except ImportError:
+    dokuwiki_available = False
+else:
+    dokuwiki_available = True
+
+
+def pytest_configure(config):
+    # A common tempdir for storing plots / PDFs and other slow test related data
+    # generated during execution.
+    tempdir_session = tempfile.mkdtemp()
+    config.option.tempdir = tempdir_session
+    html_file = tempfile.NamedTemporaryFile(delete=False)
+    # Html test report will be generated at this filepath by pytest-html plugin
+    config.option.htmlpath = html_file.name
+
+
+@remote_data
```

Comment:
and if you remove `@remote_data` it stops working?
and you have to add `--remote-data` in the commandline or it won't work?